cmake/CMakeLists.txt

```diff
@@ -1063,6 +1063,10 @@ if (WIN32)
         list(APPEND ORT_WARNING_FLAGS "/wd5054")
         # Enable warning: data member 'member' will be initialized after data member 'member2' / base class 'base_class'
         list(APPEND ORT_WARNING_FLAGS "/w15038")
+        if (onnxruntime_ENABLE_TRAINING)
+          # warning C4127: conditional expression is constant(in cmake\external\protobuf)
+          list(APPEND ORT_WARNING_FLAGS "/wd4127")
```

Comment:
We have these warnings already taken care of. Usually we create either our own header file with those warning directives or include it with warning wrappers in the code.